src/test/java/com/thealgorithms/datastructures/trees/TreeTestUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.thealgorithms.datastructures.trees;
+
+import java.util.LinkedList;
+import java.util.Queue;
+import com.thealgorithms.datastructures.trees.BinaryTree.Node;
+
+public class TreeTestUtils {
+
+    /**
+     * Creates a binary tree with given values
+     *
+     * @param values: Level order representation of tree
+     * @return Root of a binary tree
+     */
+    public static Node createTree(final Integer[] values) {
```

Comment:
```suggestion
    public static Node createBinarySearchTree(final Integer[] valuesLevelOrder) {
```

Reply:
Not necessarily. It can create any binary tree for the given level order traversal.

Reply:
My bad